packages/tools/src/utilities/getVolumeId.ts

```diff
@@ -0,0 +1,8 @@
+export const getVolumeId = (targetId: string) => {
```

Comment:
You need to check that targetId starts with volumeId first, otherwise this code does weird things on imageId substrings.

Reply:
It already does weird things on imageId that's the reason of the PR...
This PR is tiring me for a little fix of a regression introduced 3 weeks ago!!

Reply:
It's already tested anyway...

Reply:
@wayfarer3130 could you write the code that makes me working with a nifti:xxxx.com/xxx?xxx=xxx&xxx=xxx that is the volumeId I get in the project. Thanks.

Reply:
It's up to date now, could you merge it or fix the problem please?? Thanks!